### PR TITLE
feat: add view toggle menu to toolbar of attachment block

### PR DESCRIPTION
**Pull request by fundon**
Closes: [BS-1507](https://linear.app/affine-design/issue/BS-1507/attachment-card-view-添加切换为-pdf-embed-view-的功能)

### CODE REVIEW

**Comment by fundon** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/toeverything/blocksuite/8660?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#8660** <a href="https://app.graphite.dev/github/pr/toeverything/blocksuite/8660?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#8658** <a href="https://app.graphite.dev/github/pr/toeverything/blocksuite/8658?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @fundon and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: packages/blocks/src/root-block/widgets/element-toolbar/change-attachment-button.ts [hunk 1]
```diff
@@ -85,47 +80,66 @@ export class EdgelessChangeAttachmentButton extends WithDisposable(LitElement) {
     return this.edgeless.std;
   }
 
+  get viewToogleMenu() {
```
[the remark this answers is not in the document]
Comment (doodlewind) [reply packages/blocks/src/root-block/widgets/element-toolbar/change-attachment-button.ts:79]: @fundon you should fix this typo.

[the remark this answers is not in the document]
Comment (fundon) [reply packages/blocks/src/root-block/widgets/element-toolbar/change-attachment-button.ts]: fixed

**Review by L-Sun — APPROVED** [review]

**Comment by fundon** [comment]
### Merge activity

* **Nov 4, 10:20 PM EST**: The merge label 'merge' was detected. This PR will be added to the [Graphite merge queue](https://app.graphite.dev/merges?org=toeverything&repo=blocksuite) once it meets the requirements.

**Review by doodlewind — APPROVED** [review]

**Pull request merged**